src/containers/FormikForm/components/FormikAvailability.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2021-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React from 'react';
+import { RadioButtonGroup } from '@ndla/ui';
+import { injectT, tType } from '@ndla/i18n';
+import { FieldInputProps } from 'formik';
+import { AvailabilityType } from '../../../interfaces';
+
+interface Props {
+  availability: string;
+  field: FieldInputProps<string[]>;
+}
+
+const FormikAvailability = ({ availability, field, t }: Props & tType) => {
+  const availabilityValues: AvailabilityType[] = ['everyone', 'teacher', 'student'];
```

Comment:
Burde denne kansje være i en constants eller helper fil?

Reply:
Jeg ombestemte meg litt.
Er ikke availabilityValues og AvailabilityType begge noe som bare brukes i denne filen? De kunne vel begge vært definert i denne filen.

Reply:
Jeg la til `AvailabilityType` i interfaces.ts, så dersom det skulle bli brukt flere steder så er den tilgjengelig for alle. Selve AvailabilityValues er kun definert her siden det er her den blir brukt.

Reply:
Syntes ikke det er den beste løsningen
Det går ann å gjøre:
const availabilityValues = ['everyone', 'teacher', 'student'] as const;
type AvailabilityType = typeof availabilityValues[number];